### PR TITLE
C2LC-405: Add utterance rate to the AudioManager constructor

### PR DESCRIPTION
**Pull request by chosww**
See [C2LC-405](https://fluidproject.atlassian.net/browse/C2LC-405) for details.

### CODE REVIEW

**Comment by chosww** [comment]
This work has a wrong branch number for the commit message.

File: src/App.js [hunk 1]
```diff
@@ -381,6 +382,8 @@ export class App extends React.Component<AppProps, AppState> {
             isDraggingCommand: false,
             audioEnabled: true,
             announcementsEnabled: true,
+            // Keep utteranceRate in the state, which later have a controller to change
```
Comment (the-t-in-rtf) [inline src/App.js:385]: "so that we can add controls to change it later"?

**Comment by the-t-in-rtf** [comment]
I saw your comment on the ticket about Opera, the speed seems the same on the desktop version of Opera.  Did you mean mobile or another version?

**Comment by chosww** [comment]
> I saw your comment on the ticket about Opera, the speed seems the same on the desktop version of Opera. Did you mean mobile or another version?

My bad, yeah it is Opera Android that it is not supported.

File: src/App.js (hunk 1 above)
Comment (chosww) [reply src/App.js]: that sounds way more clear!

**Comment by the-t-in-rtf** [comment]
I'm happy for this to move on to a full review if you are, @chosww.

**Pull request closed without merging**